src/scenic/syntax/compiler.py

```diff
@@ -9,10 +10,34 @@
 def compileScenicAST(scenicAST: ast.AST) -> Tuple[ast.AST, List[ast.AST]]:
     """Compiles Scenic AST to Python AST"""
     compiler = ScenicToPythonTransformer()
-    node = ast.fix_missing_locations(compiler.visit(scenicAST))
+    tree = compiler.visit(scenicAST)
+    if isinstance(tree, list):
+        node = [ast.fix_missing_locations(n) for n in tree]
```

Comment:
If this can return a list, the type annotation should change.

Reply:
Fixed in 38b02303be4841a4ee19f9e54de5714d63c8ac9c